simcon_project/simcon_project/settings.py

```diff
@@ -41,7 +41,6 @@
     'bootstrap_modal_forms',
     'django_tables2',
     'embed_video',
-    'django_tables2',
```

Comment:
whoever merges first, you or david will need to make sure u didnt erase both

Reply:
I'll probably merge first, then he can pull main